packages/sit-onyx/.storybook/managed.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ArgTypesEnhancer,
+  InputType,
+  SBType,
+  StrictInputType,
+  walkTree,
+} from "@sit-onyx/storybook-utils";
+
+const SB_TYPE_CONTROL_MAP = new Map<SBType["name"], InputType["control"]>([
+  ["boolean", { type: "boolean" }],
+  ["string", { type: "text" }],
+  ["number", { type: "number" }],
+]);
+
+const getManagedParent = (inputType?: StrictInputType) => {
+  if (!inputType?.type || inputType.table?.defaultValue?.summary !== "MANAGED_SYMBOL") {
+    return undefined;
+  }
+
+  return walkTree(inputType.type, (elem, parent) =>
+    elem.name === "symbol" || (elem.name === "other" && elem.value === "unique symbol")
+      ? parent
+      : undefined,
+  );
+};
+
+export const enhanceManagedSymbol: ArgTypesEnhancer = (context) => {
+  Object.values(context.argTypes)
+    .map((argType) => {
+      const parent = getManagedParent(argType);
+      return { argType, parent };
+    })
+    .filter(({ parent }) => parent)
+    .forEach(({ argType, parent }) => {
+      const firstAvailableControl = walkTree(parent || argType.type!, (sb) =>
+        SB_TYPE_CONTROL_MAP.get(sb.name),
+      );
+
+      if (firstAvailableControl && argType.table?.defaultValue) {
+        argType.control = firstAvailableControl;
+        argType.table.defaultValue.detail =
+          "<pre>\n" +
```

Comment:
It seems like `<pre>` is not rendered as HTML so should we remove it?
![image](https://github.com/user-attachments/assets/19135a8a-cc70-44b0-9033-076eb84a37a2)

Reply:
Details are always styled as JSX :/